packages/frontend/core/src/providers/modal-provider.tsx

```diff
@@ -24,6 +24,7 @@ import { PaymentDisableModal } from '../components/affine/payment-disable';
 import { useAsyncCallback } from '../hooks/affine-async-hooks';
 import { useNavigateHelper } from '../hooks/use-navigate-helper';
 import { AuthService } from '../modules/cloud/services/auth';
+import { PeekViewManagerModal } from '../modules/peek-view/view';
```

Comment:
```suggestion
import { PeekViewManagerModal } from '../modules/peek-view';
```

always import from the root index.ts of a module

Reply:
Im not sure about this one. Maybe importing from most specific path may get rid of possible circular dependency issues